### ISSUE TITLE
date / datetime formatting

### ISSUE THREAD

**Issue by clach04** [jsonform/jsonform#447]
With:

    {
        "value": {
            "datetime": "2000-01-01 00:00:00", 
            "date": "2000-01-01"
        }, 
        "form": [
            {
                "type": "date", 
                "key": "date"
            }, 
            {
                "type": "datetime-local", 
                "key": "datetime"
            }
        ], 
        "schema": {
            "datetime": {
                "type": "string", 
                "title": "datetime"
            }, 
            "date": {
                "type": "string", 
                "title": "date"
            }
        }
    }

in Playground https://jsonform.github.io/jsonform/playground/index.html I'm seeing US date format in the form, not the ISO/ANSI format in the json. On submission it is correct but on screen it's not. Is there any control over this?

**Comment by sdetweil** [comment]
I do not see anything in particular about iso, 
but this about html5 field types

![Screenshot_20240128_191743_Chrome](https://github.com/jsonform/jsonform/assets/1498142/22cbd29e-4594-4144-ac06-ddfd532c0166)

**Comment by sdetweil** [comment]
also
![Screenshot_20240128_192433_Chrome](https://github.com/jsonform/jsonform/assets/1498142/79783699-fb77-4751-a40d-114dba9614d3)

**Comment by clach04** [comment]
Yes, those are the type I'm using. I scanned the js code briefly and couldn't see how it worked, it was like magic! I wondered if this was a bootstrap (v3) thing?
